src/linters/index.js

```diff
@@ -7,6 +7,7 @@ const Mypy = require("./mypy");
 const Prettier = require("./prettier");
 const RuboCop = require("./rubocop");
 const Stylelint = require("./stylelint");
+const swift_format = require("./swift-format");
```

Comment:
These variable names are failing the ESLint checks. Maybe they could be called something like `SwiftFormatApple` and `SwiftFormatLockwood`/`SwiftFormatCommunity`/...?

Reply:
I think I would actually prefer differences here ... we can rename it to say `AppleSwiftFormat` (or `SwiftFormatApple`, or reflect that it was originally donated by Google!) and leave the old `swiftformat` as is just so that users don't have to migrate.  If we can avoid breaking existing users for a little bit of relative ugliness, it _may_ be a reasonable tradeoff.  If you really would prefer the uniformity, I'm totally okay with that and will adjust it.

Reply:
Sorry for the late response, I somehow missed this comment. I totally agree regarding the breaking changes, I only meant variables/files and not the option names themselves. The option names could be made consistent in the next breaking release.

Reply:
Good point about Google being the original author, I wasn't aware of that. Maybe something like `SwiftFormatOfficial` would be a better name then?

Reply:
Or how about this:

- Lockwood: `swiftformat_lockwood` as option name with a `swiftformat` alias until v2 (to avoid breaking changes), `SwiftFormatLockwood` in code
- Apple/Google: `swift_format_official` as option name, `SwiftFormatOfficial` in code

Let me know what you think, I can update the code :)

Reply:
I like the suggestion.  If you could make that change and merge this, it would be really wonderful.